docs/mergeduty/merge-and-staging-instance.md

```diff
@@ -1,10 +1,42 @@
 # Staging and merging instance
 
-There is an AWS instance to run staging and merging instances so that we are fewer hops away from the hg repos.  
+There is an AWS instance to run staging and merging instances so that we are fewer hops away from the hg repos.
 
-To access it, you can either 
+To access it, you can either
 1. Create an new instance with the AMI named ```Mozilla release staging/merge machine```.  I used a M3.large instance with 32 gb space added.
-2. Use the existing instance named "Mozilla release staging/merge machine" in use1.
-You will need a pem file to access them, the key for it is the usual location.  You can connect as user ```ubuntu```.
+2. Use the existing instance named "Mozilla release staging/merge machine" in use1. See access and setup instructions below for using  existing instance.
 
-There is also a Dockerfile attached to <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1400015">1400015</a> that can be used to install the requirements for this machine, however, there is still some work to be done to address the credentials issue.
+note: There is also a Dockerfile attached to <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1400015">1400015</a> that can be used to install the requirements for this machine, however, there is still some work to be done to address the credentials issue.
+
+### Access and setup existing merge instance
+
+1. the instance may be stopped. Use the aws web console and search for Mozilla release staging/merge instance (currently i-0a18b39a0efc11658). Start the instance from the UI actions
+2. Decrypt the pem key (keys/aws-staging-mergeday.pem.gpg) from the usual location so you can ssh to the host, setup ssh forwarding and hg so you can push to hg.m.o, and clear up unneeded disk space. See below
+
+```bash
+$ # copy the public DNS from AWS console and connect through vpn+jumphost using the `ubuntu` user : e.g. ubuntu@ec2-54-83-67-250.compute-1.amazonaws.com
+$ # verify you can ssh. Then copy your mozilla ssh *public* key to the remote merge instance
+$ scp -i aws-staging-mergeday.pem ~/.ssh/${YOUR_MOZILLA_SSH_PUB} ubuntu@${DNS_OF_MERGE_INSTANCE}:~/.ssh/${USER}_rsa.pub
```

Comment:
We should puppetize this at some point.

Reply:
yep. this was just stood up quickly as a local laptop replacement. Should "production-fy" it.

Reply:
tracking: https://bugzilla.mozilla.org/show_bug.cgi?id=1441998